Add OLS-Nigeria Meetup blog post (#937)
open-life-science/open-life-science.github.io: _posts/2024-12-19-ols-nigeria-meetup.md

Needless to say, we clicked. Big time.

We kicked off our day with an **escape room challenge**—“a team-building exercise,” we told ourselves. The mission? Solve a murder case within 30 minutes. Did we crack it? Nope. (Turns out we’re better at our jobs than catching imaginary killers.) But we uncovered bigger mysteries – Taj is great at Chemistry, Jilaga’s an undercover detective, and Seun will seize every opportunity to taunt Debs by fumbling with the light switch.

Next stop: the **Discovery Museum**. It was a walk through Nigerian history, culture, and creativity. We soaked up every moment, from admiring the exhibits to singing along to iconic Nigerian throwback tunes.  A documentary on the effect of the "Boko Haram" insurgency had us reflecting deeply on the resilience of our nation, and feeling grateful for the gift of life. And, of course, we couldn’t resist snapping photos at every turn…

> ![image alt]({% link images/ols-ng-meetup/ols-ng-collage.jpg %}) 

Reviewer: but what's in the image? :) Alt text please :)

Author: All in due time, dear Yochannah.

Author: Done, done, done! ✨

> ![image alt]({% link images/ols-ng-meetup/ols-ng-collage.jpg %}) 

Author: ```suggestion
![A photo collage containing individual images, each telling a unique story. At the top left corner, the first picture shows two people sitting on a white bench in an art gallery: Seun, an adult black male wearing a blue t-shirt and gray jeans, and Debs, an adult black female with red hair extensions and a purple jumpsuit with white dots. At the top right corner, Seun is smiling in a photo with an adult black female, Jilaga, who has cornrows and dark shades. There are three images in the middle row of the collage: the first photo has Jilaga and Debs standing beside an adult black male, Taj, who is wearing a beige African "senator" outfit with glasses. They are in front of a large screen showing different Nigerian currencies. Next to this image, is a picture showing the evolution of Nigerian music, from 50s to 70s. The last image in the middle row contains traditional beads of the Igbos, and a screen displays palm kernels - above, a sign reads "Igbo culture." The third and final row of the collage has two images: on the left, Taj, Seun and Jilaga pose for a photo, and on the right, a table is filled with dishes like rice, salad and chow mein.]({% link images/ols-ng-meetup/ols-ng-collage.jpg %}) 
```

Author: Too much? @yochannah 😀

By lunchtime, the conversation flowed as freely as the food. Fun fact: there’s a short list of things Nigerians love more than good food. We shared bites of our meals, in the same way we dove into each other’s worlds. We shared laughs, swapped stories, and found common ground in our unique journeys within OLS.

We got to learn about Jilaga’s background in Architecture–mind blown! She also gave us the inside scoop on the intricate process of design and branding. 
Taj, on the other hand, spoke about his academic pursuit, which has taken him across continents, and his growing passion for Project Management and AI.
Meanwhile, Seun talked about his Bioinformatics background, his activity within the OLS and his Community’s participation in The Catalyst project. And Debs? It was a time to reflect on growth and new possibilities.

To wrap up the day on a sweet note, we made our way to a cozy spot for gelato.  It was the perfect ending to an unforgettable day—because what’s better than bonding over shared love for dessert?

> ![image alt]({% link images/ols-ng-meetup/ols-ng-gelato.jpeg %}){: style="max-width: 50%; max-height: 500px;"}

Reviewer: alt text :)

Author: Added. Thanks! 😗

> ![image alt]({% link images/ols-ng-meetup/ols-ng-gelato.jpeg %}){: style="max-width: 50%; max-height: 500px;"}

Author: ```suggestion
![Photo shows four (4) separate cups containing two (2) scoops of gelato. Each cup is placed on a white serviette.]({% link images/ols-ng-meetup/ols-ng-gelato.jpeg %}){: style="max-width: 70%; max-height: 500px;"}
```

This meet-up wasn’t just a chance to put faces to names; it was a celebration of camaraderie. We may have started the day as colleagues, but by the end of it, we were more like a family. 

So, here’s to more in-person meetups! 🥂

> ![image alt]({% link images/ols-ng-meetup/ols-ng-cheers.jpeg %}){: style="max-width: 50%; max-height: 500px; display: block; margin: 0 auto;" }

Reviewer: alt please! :)

Author: Thank you. 🤗

> ![image alt]({% link images/ols-ng-meetup/ols-ng-cheers.jpeg %}){: style="max-width: 50%; max-height: 500px; display: block; margin: 0 auto;" }

Author: ```suggestion
![Photograph shows a group of four (4) raising their glasses in a celebratory toast. The glasses are filled with water, ice cubes and a slice of lemon.]({% link images/ols-ng-meetup/ols-ng-cheers.jpeg %}){: style="max-width: 50%; max-height: 500px; display: block; margin: 0 auto;" }
```

---
layout: post
title: OLS Nigeria 
authors:
- npdebs
- jilaga
- seunolufemi123
- tajuddeen1

> image: images/ols-ng-meetup/ols-ng-group.jpg

Author: ```suggestion
image: images/ols-ng-meetup/ols-ng-group.jpeg
```

> image: images/ols-ng-meetup/ols-ng-group.jpeg

Author: ```suggestion
image: /images/ols-ng-meetup/ols-ng-group.jpeg
```

photos:
  name: Jilaga
  license: CC BY-SA 4.0
  url: 
---

> # [Title to be decided]

Author: @jilaga @seunolufemi123 @Tajuddeen1  - Thoughts on a title?

![image alt]({% link images/ols-ng-meetup/ols-ng-collage.jpg %}) 

By lunchtime, the conversation flowed as freely as the food. Fun fact: there’s a short list of things Nigerians love more than good food. We shared bites of our meals, in the same way we dove into each other’s worlds. We shared laughs, swapped stories, and found common ground in our unique journeys within OLS.

We got to learn about Jilaga’s background in Architecture–mind blown! She also gave us the inside scoop on the intricate process of design and branding. 

Taj, on the other hand, spoke about his academic pursuit, which has taken him across continents, and his growing passion for Project Management and AI.

Meanwhile, Seun talked about his Bioinformatics background, his activity within the OLS and his Community’s participation in The Catalyst project. And Debs? It was a time to reflect on growth and new possibilities.

To wrap up the day on a sweet note, we made our way to a cozy spot for gelato.  It was the perfect ending to an unforgettable day—because what’s better than bonding over shared love for dessert?

> ![image alt]({% link images/ols-ng-meetup/ols-ng-gelato.jpg %}) 

Author: ```suggestion
![image alt]({% link images/ols-ng-meetup/ols-ng-gelato.jpeg %}) 
```

This meet-up wasn’t just a chance to put faces to names; it was a celebration of camaraderie. We may have started the day as colleagues, but by the end of it, we were more like a family. 

So, here’s to more in-person meetups! 🥂

> ![image alt]({% link images/ols-ng-meetup/ols-ng-cheers.jpg %}) 

Author: ```suggestion
![image alt]({% link images/ols-ng-meetup/ols-ng-cheers.jpeg %}) 
```
